.pre-commit-config.yaml

```
@@ -0,0 +1,30 @@
+exclude: |
+  (?x)
+  # NOT INSTALLABLE ADDONS
+  # END NOT INSTALLABLE ADDONS
+  # Files and folders generated by bots, to avoid loops
+  ^setup/|/static/description/index\.html$|
+  # We don't want to mess with tool-generated files
+  .svg$|/tests/([^/]+/)?cassettes/|^.copier-answers.yml$|^.github/|
+  # Maybe reactivate this when all README files include prettier ignore tags?
+  ^README\.md$|
+  # Library files can have extraneous formatting (even minimized)
+  /static/(src/)?lib/|
+  # Repos using Sphinx to generate docs don't need prettying
+  ^docs/_templates/.*\.html$|
+  # You don't usually want a bot to modify your legal texts
+  (LICENSE.*|COPYING.*)
+default_language_version:
+  python: python3
+  node: "14.18.0"
+repos:
+  - repo: https://github.com/acsone/setuptools-odoo
+    rev: 3.0.3
+    hooks:
+      - id: setuptools-odoo-make-default
+      - id: setuptools-odoo-get-requirements
+        args:
+          - --output
+          - requirements.txt
+          - --header
+          - "# generated from manifests external_dependencies"
```

```suggestion
repos:
  - repo: https://github.com/acsone/setuptools-odoo
    rev: 3.0.3
    hooks:
      - id: setuptools-odoo-make-default
      - id: setuptools-odoo-get-requirements
        args:
          - --output
          - requirements.txt
          - --header
          - "# generated from manifests external_dependencies"

```
Wasn't sure default_language_version section is needed.  Can you please check and add it back in as necessary.